test/e2e/multi-cluster-deployment/run-test.sh

```diff
@@ -16,8 +16,9 @@
 # This is an end to end test of multi cluster deployement.  
 # For readable instructions, please visit https://github.com/kubestellar/kubestellar/tree/ks-0.20/docs/content/v0.20
 
+set -x # so users can see what is going on
 set -e # exit on error
 
+./cleanup.sh
```

Comment:
I am a bit conflicted here.  On the one hand, I personally love having a cleanup phase before the test.  On the other hand, this test case might be running against an existing kind cluster that have just been used for some other testing and it won't be right to remove them.  I won't object if you feel strongly about this change.  But let me list yet a third option - check for existence of needed clusters and fail the test here if pre-existing with a message recommending the user runs cleanup.sh.  I leave this in your very capable hands.

Reply:
This test _is_ destructive of the clusters involved. I have never heard anybody complain about a test destroying clusters that they wanted to preserve. OTOH, I have heard many complaints about tests failing due to conflict with pre-existing state. I suggest we start with this forceful initial cleanup, and if we get complaints then we can later introduce a choice between (a) forceful cleanup or (b) pre-flight checks with good error messages.